### PR TITLE
ModelViewSet for accounts 

### PR DESCRIPTION
**Pull request by theyashshahs**
It contains event and registration app for the user 

-----
[View rendered README.md](https://github.com/yashshah2820/cerebro-backend-2020/blob/registration/README.md)

### CODE REVIEW

File: accounts/views.py [hunk 1]
```diff
@@ -34,3 +36,29 @@ def get_queryset(self):
         return Account.objects.all()
 
 
+class AccountCreateView(generics.CreateAPIView):
```
Comment (Monal5031) [inline accounts/views.py]: Again, use ModelViewSet for list and CRUD

File: registration/models.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Dashboard(models.Model):
```
Comment (Monal5031) [inline registration/models.py:4]: ...?

File: accounts/models.py [hunk 3]
```diff
@@ -36,3 +37,11 @@ def email_user(self, subject, message, from_email=None, **kwargs):
     def __str__(self):
         return self.first_name + ' ' + self.last_name + ' (' + self.email + ')'
 
+
+class GoogleAccount(models.Model):
```
Comment (Monal5031) [inline accounts/models.py]: Why are we creating a separate Model for this? There are Oauth backends available in `python-social-core`

**Review by Monal5031 — CHANGES_REQUESTED** [review]

File: registration/models.py (hunk 2 above)
Comment (theyashshahs) [reply registration/models.py:4]: for the events, the user will register for

File: accounts/models.py (hunk 3 above)
Comment (theyashshahs) [reply accounts/models.py]: okay sir

File: accounts/views.py (hunk 1 above)
Comment (theyashshahs) [reply accounts/views.py]: it's below sir after testing I will remove generics

**Pull request closed without merging**